misc/prototypes.h

```diff
@@ -44,6 +44,45 @@ void *mmap64(void *addr, size_t length, enum uft_mmap_prot prot, enum uft_mmap_f
 int munmap(void *addr, size_t length);
 int mprotect(void *addr, size_t len, enum uft_mmap_prot prot);
 
+enum uft_madvise_advice {
+    MADV_NORMAL      = 0,
+    MADV_RANDOM      = 1,
+    MADV_SEQUENTIAL  = 2,
+    MADV_WILLNEED    = 3,
+    MADV_DONTNEED    = 4,
+    MADV_FREE        = 8,
+    MADV_REMOVE      = 9,
+    MADV_DONTFORK    = 10,
+    MADV_DOFORK      = 11,
+    MADV_MERGEABLE   = 12,
+    MADV_UNMERGEABLE = 13,
+    MADV_HUGEPAGE    = 14,
+    MADV_NOHUGEPAGE  = 15,
+    MADV_DONTDUMP    = 16,
+    MADV_DODUMP      = 17,
+    MADV_HWPOISON    = 100,
+};
+int madvise(void *addr, size_t length, enum uft_madvise_advice advice);
+
+enum uft_posix_madvise_advice {
```

Comment:
Sorry for bothering you, but the name seems too long.  What about `uft_posix_madvise` or `uft_posix_madvise_adv`?

Reply:
Ok. I will make it as `uft_posix_madvise_adv` then.

Reply:
Just updated it. Thanks!